modules/shortcodes/polldaddy.php

```diff
@@ -573,7 +573,7 @@ function polldaddy_link( $content ) {
 	add_filter( 'the_content_rss', 'polldaddy_link', 1 );
 
 	/** This filter is documented in modules/shortcodes/youtube.php */
-	if ( apply_filters( 'jetpack_comments_allow_oembed', get_option( 'embed_autourls' ) ) ) {
+	if ( apply_filters( 'jetpack_comments_allow_oembed', true ) ) {
```

Comment:
I think I understand this a bit better now. WordPress didn't remove the option, exactly, it removed the UI.

WordPress uses the `default_option_` filter (as opposed to the `pre_option_` or `option_` filters).  That means if someone has the option set to some value in the database, `get_option( 'embed_autourls' )` will return that value (for example, `false`).

So removing the option call breaks backward compatibility in two ways:
1. People could be using one of the options filters to adjust this behavior, and
2. People could have a value set in the database to adjust this behavior.

I'm not against breaking backward compatibility here - I just want to make sure we know that we're doing it.  It's probably very rare that anyone is depending on either of those behaviors.